onboarding_documentation/ingestion/ingestion_tutorial.md

```diff
@@ -0,0 +1,218 @@
+    >
+    > We will be generating metadata in four different tables in metamist from each row in the metadata manifest file. We will create
+        >
+        > A new participant in the `Participant` table (csv column: `Individual ID`)
+        > A new sample in the `Sample` table linked to the relevant participant (csv column: `Sample ID`)
+        > A new assay in the `Assay` table linked to the relevant sample and sequencing group (csv column: `Filenames`)
+        > A new sequencing group in the `Sequencing Group` table that links related assays together
+    >
+    > The image below captures the relationship between the different elements of the metadata that we will be ingesting in Metamist.
+    >
+    > ![Image of the relationship between metdata elements in metamist](../technical_documentation/metamist_docker/sequencing_group_relationships.png)
+    >
+    > An up-to-date Metamist schema is available in the metamist repository. Not all fields in the metadata schema are mandatory and metadata is ingested differently for each table.
+    >
+    > _We are adding new metadata to Metamist in this tutorial. When metadata already exist in Metamist, the process is different and care must be taken to avoid overwriting records._
+    >
+
+
+4. Find an existing CPG parser to ingest your metadata
+
+    Parsing first checks to see if the metadata being ingested conform to an anticipated structure. Then parsing will transform the metadata listed in the metadata manifest to populate the required tables in Metamist.
+
+    Ingestion is normally performed at the CPG using existing software modules called Parsers. These are written in Python and are maintained jointly by Software and Analysis teams. Parsers are located in `metamist/parser`. Parsers are designed using [class-based inheritance](https://en.wikipedia.org/wiki/Inheritance_(object-oriented_programming)), which adapts generic parsers to custom use cases like different datasets.
+
+    We will use the parser `metamist/parser/sample_file_map_parser.py` for this tutorial, which is located in the Metamist repository. Take a moment to read the comments at the head of the file, but know that there’s no need to understand this exhaustively.
+
+5. Ensure that the metadata manifest is accessible to the parser
+
+    If the metadata manifest is co-located with your data in the Google Clouds storage bucket, you do not need to take any action. This will be accessible to the parser.
+
+    If your collaborator has provided the metadata manifest separately to the data, please download the manifest to your local machine and add it to a new folder in your local metamist repository. Add this folder to your .gitignore file to ensure that this change is not pushed to the remote repository.
```

Comment:
The manifest could be stored anywhere, it needn't be saved to the local repo. It could be uploaded to the bucket to promote that co-location with the data files. Best practices would probably involve not saving it to the local repo at all, so maybe we shouldn't suggest that here. Even though risks are mitigated by adding the manifest path to the .gitignore file, I think we should still steer away from suggesting the manifest should be saved to the user's local repo.

Reply:
Good point. I've reduced this paragraph and encouraged the user to upload their manifest to the same bucket as their data.